src/ConfigManager.ts

```diff
@@ -18,21 +18,27 @@ type ProxyHostConfig = {
   proxyUseCustomMethod: string
   timeoutSeconds: number
   stopOnTimeoutIfCpuUsageBelow?: number
+  disableDefaultLoadingPage?: boolean
+  enableDefaultLoadingPage?: boolean
+  customHttpStatusReadyChecking?: number
```

Comment:
Please update the README to include the new options and a short explanation on how and when to use them